faucet/frontend/components/common/TwitterSignIn/index.jsx

```diff
@@ -0,0 +1,93 @@
+import Button from '../Button';
+import axios from 'axios';
+import { decode } from 'jsonwebtoken';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+
+const TwitterSignIn = function ({
+	twitterAccountStatus,
+	setTwitterAccountStatus,
+	authUrl
+}) {
+	const [isLoading, setIsLoading] = useState(false);
+
+	const authRequest = axios.create();
+	authRequest.defaults.baseURL = authUrl;
+
+	const twitterAuth = async () => {
+		setIsLoading(true);
+		const { data } = await authRequest.get('/twitter/auth');
+		localStorage.setItem('twitterOauthTokenSecret', data.oauthTokenSecret);
```

Comment:
is there a reason you use localStorage vs sessionStorage?
https://developer.twitter.com/ja/docs/basics/authentication/guides/securing-keys-and-tokens

Reply:
actually nope, but it seems I should use sessionStorage rather than localStorage.

Reply:
I have made the PR here:
https://github.com/symbol/faucet-repo/pull/72